docs/examples/metrics/grocery/grocery.py

```diff
@@ -0,0 +1,115 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from math import inf
+
+from opentelemetry.sdk.metrics.export import ConsoleExporter
+from opentelemetry.metrics import get_meter_provider
+
+
+exporter = ConsoleExporter()
+
+get_meter_provider().start_pipeline_equivalent(exporter, 5)
```

Comment:
Curious - would you try to put the configuration part after the `GroceryStore` class and see if the global meter provider would work for delay loading?

For example:

```python
meter = get_meter_provider().get_meter()
order_counter = meter.create_counter(...)
amount_counter = meter.create_counter(...)

class GroceryStore:
    ....


if __name__ == '__main__':
    exporter = ConsoleExporter()
    # configure the global meter provider and exporter
```

Reply:
Sorry, what do you mean with delay loading?

Reply:
This file has been removed, resolving.